### PR TITLE
stat: new version 4.1.0

### PR DESCRIPTION
**Pull request by lee218llnl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/stat/package.py [hunk 1]
```diff
@@ -48,10 +51,11 @@ class Stat(AutotoolsPackage):
     # we depend on mpa@master for bug fixes since launchmon 1.0.2
     depends_on('launchmon@master')
     depends_on('mrnet')
+    depends_on('python')
     depends_on('python@:2.8', when='@:4.0.0')
     depends_on('py-pygtk', type=('build', 'run'), when='@:4.0.0')
     depends_on('py-enum34', type=('run'), when='@:4.0.0')
-    depends_on('py-xdot@1.0', when='@4.0.1:')
+    depends_on('py-xdot@1.0', when='@4.0.1: +gui')
```
Comment (sethrj) [inline var/spack/repos/builtin/packages/stat/package.py:58]: Doe py-gtk needed a new `+gui` qualifier as well?

**Review by tldahlgren — APPROVED** [review]
Confirmed new version available at the URL.

**Review by sethrj — COMMENTED** [review]
Whoops, forgot to click "submit review"... sorry.

File: var/spack/repos/builtin/packages/stat/package.py (hunk 1 above)
Comment (lee218llnl) [reply var/spack/repos/builtin/packages/stat/package.py:58]: Thanks for pointing this out. A better option may be to deprecate/delete older versions of STAT to clean up the package. I may take that as a separate to-do. In the meantime, I don't think anyone will be building an older version, so I don't think there is any harm in leaving it as is for now.

Comment (sethrj) [reply var/spack/repos/builtin/packages/stat/package.py:58]: Agreed, and sorry about the unintentional post-merge review! 😂

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/stat/package.py:58]: > Agreed, and sorry about the unintentional post-merge review! 😂

Apologies for being "too quick" on the review/merge.

Comment (lee218llnl) [reply var/spack/repos/builtin/packages/stat/package.py:58]: please see #22650

**Pull request merged**